std_msgs/Header header

uint8[3] zone_occupied
uint32 zone_config_id